addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -520,7 +520,7 @@ export class Wysiwyg extends Component {
         });
 
         if (options.snippets) {
-            $(this.odooEditor.document.body).addClass('editor_enable');
+            $(this.odooEditor.document.body).addClass('editor_enable o_handlers_idle');
```

Comment:
Instead of having to touch this line, could not you reverse the logic? (adding a class during the time it is not "idle"). Also could you not add it on the overlay instead of the body?

Reply:
This was a solution we found with @stefanorigano (https://github.com/odoo/odoo/pull/132653#discussion_r1312895822) to avoid "ugly" css selector like this one : 
```css
 body:not([class$="-important"]) &:hover, &:active {
 ```
 What do you think ?

Reply:
This was indeed ugly but I think my two points remain the same:
- Right now you have: "there are styles for when o_handlers_idle is there and there are styles for o_handlers_idle is not there", you could just invert that logic: so in your example, you would not have `body:not([class$="-important"])` but `body:not(.o_handlers_enabled)` (something like that)
- If I am not mistaken, your style which depends on this `o_handlers_idle` class only affects the overlay: so could not the class be toggled on the overlay itself rather than on the body?

Reply:
> "there are styles for when o_handlers_idle is there and there are styles for o_handlers_idle is not there"

There are styles on hover **only** when `o_handlers_idle` is there. The goal is to prevent `:hover` styles from other handles when the user is already dragging a handle. So I guess this logic makes sens, or did I missed something ? 🤔 

Done, I moved the class into the overlay 👍